pkg/util/iptables/iptables_test.go

```diff
@@ -1198,3 +1198,53 @@ func TestRestoreAllWaitBackportedIptablesRestore(t *testing.T) {
 		t.Errorf("expected failure")
 	}
 }
+
+// TestExtractLines tests that
+func TestExtractLines(t *testing.T) {
+	mkLines := func(lines ...LineData) []LineData {
+		return lines
+	}
+	lines := "Line1: 1\nLine2: 2\nLine3: 3\nLine4: 4\nLine5: 5\nLine6: 6\nLine7: 7\nLine8: 8\nLine9: 9\nLine10: 10"
+	tests := []struct {
+		count int
+		line  int
+		name  string
+		want  []LineData
+		lines []byte
+	}{{
+		name:  "test-line-0",
+		count: 3,
+		line:  0,
+		lines: []byte(lines),
```

Comment:
this is the same for every case - you can just leave it out and say it in the test loop at the bottom :)

Reply:
done